### PR TITLE
Implemented family and type functionality for creating RoutingKey in REMREM Semantics

### PR DESCRIPTION
**Pull request by xumakap**

### CODE REVIEW

File: src/main/java/com/ericsson/eiffel/semantics/events/Event.java [hunk 1]
```diff
@@ -16,4 +18,24 @@ public void generateMeta(String msgType, JsonObject msgNodes) {
         meta.setId(UUID.randomUUID().toString());
     }
     public abstract void setMeta(Meta meta);
+
+    public static String getFamilyRoutingKey(String eventType) {
+        RoutingKeyTypes eventtype = null;
+        try {
+            eventtype = RoutingKeyTypes.valueOf(eventType);
```
Comment (xmairaa) [inline src/main/java/com/ericsson/eiffel/semantics/events/Event.java]: It should return RoutingKeyTypes.valueOf(eventType).getFamily()

File: src/main/java/com/ericsson/eiffel/semantics/events/Event.java [hunk 2]
```diff
@@ -16,4 +18,24 @@ public void generateMeta(String msgType, JsonObject msgNodes) {
...
+        } catch (Exception e) {
+            log.error("Unknown event type requested: " + eventType);
+        }
+        return eventtype.getFamily();
+    }
+
+    public static String getTypeRoutingKey(String eventType) {
+        RoutingKeyTypes eventtype = null;
+        try {
+            eventtype = RoutingKeyTypes.valueOf(eventType);
```
Comment (xmairaa) [inline src/main/java/com/ericsson/eiffel/semantics/events/Event.java]: Same as my above comment

File: src/main/java/com/ericsson/eiffel/semantics/events/Event.java (hunk 1 above)
Comment (xmairaa) [inline src/main/java/com/ericsson/eiffel/semantics/events/Event.java]: It should return RoutingKeyTypes.valueOf(eventType).getFamily()

File: src/main/java/com/ericsson/eiffel/semantics/events/Event.java [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 import java.util.UUID;
+
+import com.ericsson.eiffel.remrem.semantics.RoutingKeyTypes;
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
 
 public abstract class Event {
     public transient Meta meta;
-
+    public static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(Event.class);
```
Comment (xmairaa) [inline src/main/java/com/ericsson/eiffel/semantics/events/Event.java]: use import for the package org.slf4j.

**Comment by l-ohlen-ericsson** [comment]
It looks like these 3 keys are missing in the EiffelConstants.java file
"info"
"configuration"
"flowcontext"

**Pull request merged**